### ISSUE TITLE
close

### ISSUE THREAD

**Issue by spacct** [Atlantiss/NetherwingBugtracker#8727]

**Comment by Healweed** [comment]
I was about to make the same bugreport until i saw this now.
I'ma just gonna put on a more detail of all 3 trinkets

Netherwing Reputation/Shadowmoon Valley:
Overseer Badge(Friendly Reputation Trinket):
Does Not melee attack, only follows the target in melee range
Does not cast Arcane Bolt

Captain's Badge(Honored Reputation Trinket):
Does not melee attack, only follows the target in melee range
Does not cast Arcane Bolt

Commander's Badge:
When summoned it will occur two situations:
-If attack the current target and you use the trinket after, it will NOT attack your current target. Same situation if you summon it during midfight
-If you use the trinket and if the enemy target attacks first the whelp, then the whelp will attack and behave properly doing melee attack into arcane bolt

Basically tl,dr the most effective way is summoning the whelp(with commander's badge ofc) and going stealth either with a rogue or just druid.
